include/picongpu/plugins/IsaacPlugin.x.cpp

```diff
@@ -862,17 +862,20 @@ namespace alpaka
 {
 
     template<>
+    requires(!mp_empty<typename picongpu::isaacP::IsaacPlugin::SourceList>::value)
     struct IsKernelArgumentTriviallyCopyable<typename picongpu::isaacP::IsaacPlugin::SourceList> : std::true_type
```

Comment:
```suggestion
    template<>
    struct IsKernelArgumentTriviallyCopyable<
    picongpu::isaacP::IsaacPlugin::SourceList
> : std::bool_constant<!mp_empty<picongpu::isaacP::IsaacPlugin::SourceList>::value> {};
```
I think the problem is that template specializations cant have requires clauses

Reply:
I pushed a new version. IMO this should fix it.